tests/wasm/import_class.rs

```diff
@@ -158,7 +158,7 @@ fn switch_methods() {
 
     assert!(!switch_methods_called());
     SwitchMethods::new().b();
-    assert!(switch_methods_called());
+    assert!(!switch_methods_called());
```

Comment:
Note that this is indicative of that there are actually a few subtle behavior changes in this PR. Some things aren't quite as "final" as they used to be, although I suspect that this is ok in practice basically everywhere.